orbstation/code/_global/global_procs.dm

```diff
@@ -0,0 +1,14 @@
+// Initializes the global language list. Used in a couple different places. Named this way for future-proofing.
+/proc/orb_init_languages()
```

Comment:
I would rather name this `setup_language_list` or something like that, there doesn't seem to be a strong reason to prename it with `orb`

Reply:
Good point. I was worried someone would make a proc called "init_languages()" down the line so I used one I knew would never appear upstream. But this is probably fine.